### ISSUE TITLE
correct screenshot alt text and hosting

### ISSUE THREAD

**Issue by hazel-nut** [digitalocean/navigators-guide#36]
we shouldn't host screenshots on imgur; they should live on a DO assets server

all screenshots should have descriptive alt text
